router/usecase/quote.go

```diff
@@ -13,6 +13,16 @@ import (
 	"github.com/osmosis-labs/osmosis/osmomath"
 )
 
+var (
+	one = osmomath.OneDec()
+)
+
+var (
+	_ domain.Quote = &quoteImpl{}
```

Comment:
Let's rename this to `quoteExactAmountIn`

Reply:
To be honest initially I had the same idea, but decided not to do that to avoid making PR larger. It's updated now.